sdk/src/test/java/com/mapbox/maps/MapboxMapTest.kt

```diff
@@ -1068,19 +1104,24 @@ class MapboxMapTest {
   }
 
   @Test
-  fun setStyleTransition() {
+  fun setsStyleTransitionAfterOnStyleDataEvent() {
```

Comment:
Do we have tests with fully inlined style json? For example, style without sprites / fonts. Only inlined source + style + runtime modifications?

Reply:
Checked it with style with json without sprites/fonts - indeed, we only receive `StyleDataType.STYLE` + `StyleDataType.SOURCES`, no `StyleDataType.SPRITE` event before `onStyleLoad`. 
Will update the logic to make sure we load everything not yet loaded on the final `onStyleLoad` callback.

Reply:
Moreover - style without sources won't send `StyleDataType.SOURCES` either.

Reply:
For now I've changed sources, layers and images `bind` calls to happen in the final onStyleLoad event to release it today. For the next beta will update it further to respect the optional SOURCES/SPRITES event correctly.